lib/package-managers/npm.js

```diff
@@ -8,34 +8,18 @@ exports.NPM = class {
     this.executableName = 'npm';
   }
 
-  install(packages, workingDirectory) {
```

Comment:
Please leave the npm.js refactor to a separated PR. I guess you are trying to fix the hidden output from npm.
But this destroyed the yarn.js which extends npm.js, you need to test both npm and yarn.

Reply:
No this was done because npm install downloads cypress and extracts the archive but in the old setup each update of progress outputs as a single line which causes maximum buffer exception which breaks the install and you have to manually remove the package and run npm install. But yes as a side effect this also fixes the hidden output.

Reply:
Ok, then you have to make sure yarn works. The previous installCommand method is to give yarn a chance to overwrite the command format. You could just get the old installCommand back.

npm
```sh
npm i # for install all
npm i a b c # for install packages
```

yarn
```sh
yarn # for install all
yarn add a b c # for install packages
```

Reply:
But my change was only applied for npm, yarn.js is still intact with the original install method. I assumed changing the npm.js would not affect yarn in any way.

Reply:
```js
exports.Yarn = class extends npm
```

That's where it broke, your new implementation will execute `yarn i a b c`.

Reply:
ugh that's very ugly and I wish this was written using typescript 😭 
I will have a look how to fix this.

Reply:
Not the prettiest implementation I could have done, moved the original setup into the Yarn class. Not allot being shared anymore. I think I should have a look in a separate PR and see if I can refactor this into a nicer setup.